lib/plugins/backup.rb

```diff
@@ -0,0 +1,124 @@
+require 'rubygems/package'
+require 'zlib'
+require_relative '../moonshot/creds_helper'
+
+module Moonshot::Plugins # rubocop:disable Documentation
+  # Monkeypatching TarWriter for our
```

Comment:
"for our needs" is vague, can we spell out what the original `TarWriter#add` implementation does that is undesirable and how we change it?